bigtable/google/cloud/bigtable/instance.py

```diff
@@ -27,6 +27,8 @@
 
 from google.api_core.exceptions import NotFound
 
+from google.cloud.iam import Policy
```

Comment:
If adding `policy.py` this will change to 
```python
from google.cloud.bigtable.policy import Policy
```

Reply:
Done.